noxfile.py

```diff
@@ -87,9 +87,10 @@ def benchmark_tuned(session):
 
 @session
 def format(session):
-    """Runs black"""
+    """Runs black and isort"""
     session.run_always("poetry", "install", external=True)
     session.run("black", "aerleon", "tests")
```

Comment:
why is this scoped to just these two directories?

Reply:
Frankly this does need a little attention. Want to change it to target "."?

Reply:
This is probably why i had more changes since I also formatted the tools directory.    It shouldn't hurt to change this to '.'.